src/DataViews/include/DataViews/CompareAscendingOrDescending.h

```diff
@@ -2,16 +2,16 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#ifndef DATA_VIEWS_COMPARE_ASCENDING_OR_DESCENDING_H_
-#define DATA_VIEWS_COMPARE_ASCENDING_OR_DESCENDING_H_
+#ifndef DATA_VIEWS_INTERNAL_COMPARE_ASCENDING_OR_DESCENDING_H_
```

Comment:
Should this be `DATA_VIEWS_INTERNAL_COMPARE_ASCENDING_OR_DESCENDING_H_`, as the format is 
`<PATH>_<FILE>_H_` ([go/cstyle#The__define_Guard](go/cstyle#The__define_Guard))

Reply:
"internal" is not part of the directory name. This should be `DATA_VIEWS_COMPARE_ASCENDING_OR_DESCENDING_H_` like before.